src/main/resources/view/MainWindow.fxml

```diff
@@ -35,47 +35,39 @@
           </Menu>
         </MenuBar>
 
-        <HBox prefWidth="400" style="-fx-background-color: #edede9">
-          <VBox styleClass="stack-pane">
-            <AnchorPane styleClass="split-pane">
-              <ImageView fx:id="logoImage" fitWidth="100.0" pickOnBounds="true" preserveRatio="true" AnchorPane.bottomAnchor="10.0" AnchorPane.leftAnchor="50.0" AnchorPane.rightAnchor="50.0" AnchorPane.topAnchor="10.0" />
-            </AnchorPane>
-            <StackPane fx:id="piechartPlaceholder" prefHeight="400" prefWidth="200.0" styleClass="split-pane" />
-            <StackPane fx:id="statisticsPlaceholder" prefHeight="400" prefWidth="200.0" styleClass="split-pane" />
+        <HBox HBox.hgrow="ALWAYS" prefWidth="400"  style="-fx-background-color: #f1e5e5">
```

Comment:
Leaving this comment here as I can't comment on unedited portions.

The current menu bar design seems quite incongruous with the rest of the application, in terms of: font size, background colour (not sure if I'm seeing correctly but it seems to have a slight gradient), width.